pkg/apis/operator/v1/types.go

```diff
@@ -92,6 +93,11 @@ type InstallationSpec struct {
 	// kubernetesProvider.
 	// +optional
 	FlexVolumePath string `json:"flexVolumePath,omitempty"`
+
+	// NodeUpdateStrategy can be used to customize the desired update strategy, such as the MaxUnavailable
+	// field (which defaults to a value of one).
```

Comment:
I don't think we need to / should call out the default for a sub-field on the top-level struct.